src/redis-cli.c

```diff
@@ -2254,6 +2299,10 @@ static void repl(void) {
                 config.conn_info.hostport = atoi(argv[2]);
                 cliRefreshPrompt();
                 cliConnect(CC_FORCE);
+            } else if (argc == 2 && !strcasecmp(argv[0],"connect") &&
+                !strcasecmp(argv[1],"master"))
```

Comment:
as you suggested, let's also add a `connect replica` feature.
i guess these two can share code (same function that takes an argument)